src/quo/components/community/community_detail_token_gating/view.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo.components.community.community-detail-token-gating.view
+  (:require [clojure.string :as string]
+            [quo.components.community.community-detail-token-gating.style :as style]
+            [quo.components.dividers.divider-label.view :as divider-label]
+            [quo.components.markdown.text :as text]
+            [quo.components.tags.collectible-tag.view :as collectible-tag]
+            [quo.components.tags.token-tag.view :as token-tag]
+            [react-native.core :as rn]
+            [utils.i18n :as i18n]))
+
+(defn- token-view
+  [{:keys [collectible? img-src amount sufficient?] :as token}]
+  (if collectible?
+    [collectible-tag/view
+     {:collectible-name    (:symbol token)
+      :size                :size-24
+      :collectible-img-src img-src
+      :options             (when sufficient?
+                             :hold)}]
+    [token-tag/view
+     {:token-symbol  (:symbol token)
+      :size          :size-24
+      :token-value   amount
+      :token-img-src img-src
+      :options       (when sufficient? :hold)}]))
+
+(defn- tokens-row
+  [{:keys [tokens divider? first?]}]
+  [:<>
+   [rn/view
+    {:style (style/token-row first?)}
+    (map-indexed (fn [token-index token]
+                   ^{:key (str "token-" token-index)}
+                   [token-view token])
+                 tokens)]
+   (when-not divider?
+     [divider-label/view
+      {:container-style {:padding-left 0}}
```

Comment:
Do we need this zero value?

Reply:
The `divider-label` component has a default left padding of `20px`, but according to the design in this situation, we don't need any padding. That's why I'm overriding it by passing it in the container style.

Reply:
It's probably better to do the reverse?
i.e default is 0 and the other uses have 20 set

Reply:
> It's probably better to do the reverse? i.e default is 0 and the other uses have 20 set

That might involve changing it in a number of places I think

Reply:
The divider component design has left padding. For this screen only it is being removed. I think it is better to keep the component aligned with the design system.

Design: https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=771-26244&mode=dev